lldb/source/Target/Target.cpp

```diff
@@ -2318,35 +2317,38 @@ Target::GetScratchTypeSystemForLanguage(lldb::LanguageType language,
               errs->Flush();
             }
           }
-        }
```

Comment:
Readers: This closing brace being moved (to the end of the diff hunk) is the key change. The other changes in this file are formatting/layout only.